packages/atomic/src/components/product-listing/atomic-product-listing-interface/atomic-product-listing-interface.tsx

```diff
@@ -246,17 +255,13 @@ export class AtomicProductListingInterface
       return;
     }
 
-    if (!this.url) {
-      console.error(
-        'You need to set a url to fetch a product listing.',
-        this.url
-      );
-      return;
+    if (this.url) {
+      console.warn('url was manually set to :', this.url);
```

Comment:
Okay, it's useful when you are working on localhost, for example ?

And so it would still be useful for any developer trying to implement/develop their solution (not just for our own demo) ?

And those dev would need to do some changes before going in production.

I think we should beef up the warning then, be more verbose about all of this.

If you put yourself in the show of a random developer working on their Coveo project, I'm not sure that the message `url was manually set to XYZ` would help them understand what's the context.

Reply:
That's exactly the use cases where it would be needed.

I'll write a more meaningful warning.